sensors/pressure.py

```diff
@@ -1,11 +1,47 @@
 from multithreading.stream_reader import StreamReader
+import busio
+import digitalio
+import board
+import adafruit_mcp3xxx.mcp3008 as MCP
+from adafruit_mcp3xxx.analog_in import AnalogIn
+
+possible_channels = [MCP.P0, MCP.P1, MCP.P2, MCP.P3, MCP.P4, MCP.P5]
+
+
+class Manometers(StreamReader):
+    MAX_ANALOG_VALUE = 65535.0
```

Comment:
put your constants in a constant file and import them

Reply:
ça dépend, si ce sont des constantes propres au module (fichier) qui n'ont à faire qu'à cet endroit seulement, on peut les laisser. Par contre, si ces constantes sont utiles dans le même subfolder (package), on peut les mettre dans le __init__.py avec les imports

Reply:
Cette constante s'applique juste aux capteurs qui utilisent le ADC. Pour l'instant on a juste le manomètre, donc on n'a pas fait de classe comme c'est le cas avec `SerialStreamReader` ou `SMBusStreamReader`. Si c'était le cas, je bougerais ces constantes dans la classe abstraite (ex. `AnalogStreamReader`).

Pour l'instant, je vais changer la valeur à 0xFFFF, car le ADC envoie des valeurs sur un entier 16 bits, histoire d'être plus explicite.